core/src/test/java/org/elasticsearch/action/update/UpdateRequestTests.java

```diff
@@ -49,7 +49,7 @@ public void testUpdateRequest() throws Exception {
         UpdateRequest request = new UpdateRequest("test", "type", "1");
         // simple script
         request.source(XContentFactory.jsonBuilder().startObject()
-                .field("script", "script1")
+                .startObject("script").field("inline", "script1").endObject()
```

Comment:
Although this change is correct it is not necessary. An inline script with no parameters can have two forms:

```
"script": "scirpt contents"
```

or:

```
"script": {
  "inline": "script contents"
}
```